exonum-java-binding/core/rust/src/testkit/mod.rs

```diff
@@ -102,6 +102,24 @@ pub extern "system" fn Java_com_exonum_binding_testkit_TestKit_nativeFreeTestKit
     drop_handle::<TestKit>(&env, handle)
 }
 
+/// Polls events out of event queue. This forces messages handling, e.g. adding transactions
+/// to the pool.
+///
+/// This method is called internally by `createBlock` and `createBlockWithTransactions` methods.
```

Comment:
I don't see it being called. It, being a native Java method, must be suppossed to be called by some Java methods, mustn't it?

@vitvakatu , @MakarovS , 
Also, is it _required_ on the Java side? Is it possible to redefine some existing _native_ testkit methods so that _they_ ensure that these events gets processed when needed? If so, is it always sensible to do in these native methods? Or is it closer to the Java responsibility?

Reply:
> I don't see it being called. It, being a native Java method, must be suppossed to be called by some Java methods, mustn't it?

I'll add a native method to TestKit after we merge this.

Reply:
> Also, is it required on the Java side? Is it possible to redefine some existing native testkit methods so that they ensure that these events gets processed when needed? If so, is it always sensible to do in these native methods? Or is it closer to the Java responsibility?

I think we could avoid Java dealing with this and just pull events in `Java_com_exonum_binding_testkit_TestKit_nativeCreateSnapshot`. @vitvakatu what do you think?

Reply:
I.e., poll events before creating a snapshot so that the snapshot includes all the submitted transactions?

Reply:
Yes, exactly